src/reference/interactive/index.pug

```diff
@@ -86,62 +88,67 @@ block reference
+    p The #[a(href='#the-gameclient') Game Client] can change the scene a participant sees by updating the #[a(href='#groups') group] they belong to.
 
     h3#groups Groups
-    p Groups are a way to group individual participants together. #[a(href='#participants') Participants] within a group all see the same #[a(href='#scenes') Scene] and can contribute input to the controls on that scene. #[a(href='#the-gameclient') Game Clients] can create and update groups at any time, including changing which scene the group is set to.
+    p Individual participants can be grouped together using the Groups functionality. Participants within a group all see the same Scene and contribute input to the controls that are part of that scene. #[a(href='#the-gameclient') Game Clients] can create and update groups any time, including changing to the scene that the group is set to see.
 
-    p Groups can be used to create team-based experiences where groups compete to achieve a goal within the experience. For example, say a broadcaster is playing an adventure game. Interactive developers could create a "Helper" group and a "Hinderer" group. The help team would be able to heal the broadcaster and grant them buffs. The Hinderer team could spawn traps or monsters to try and get in the broadcasters way.
+    p Groups can be used to create team-based experiences where groups compete to achieve a goal within the experience. Using an adventure game again as an example, you could create a "Allies" group and a "Adversaries" group.
+    The ”Allies” group is provided with UI controls that gives them the ability to heal the character that the broadcaster is controlling or grant buffs to increase their chance of winning. Alternatively, the ”Adversaries” group could spawn traps or monsters to try to get in the way of winning using their controls.
 
-    p All participants start out in the default group. #[a(href='#the-gameclient') Game Clients] can re-assign participants to any group from the #[a(href='#the-gameclient') Game Client].
+    p Although all participants start out in the default group, #[a(href='#the-gameclient') Game Clients] can always re-assign participants into any group.
 
     h3#controls Controls
-    p A Control is an interactive component within a scene. #[a(href='#participants') Participants] can individually interact with the control, usually with their keyboard, mouse or controller. Interactive currently has two types of controls, but additional types may be added in the future. These two controls are Buttons and Joysticks.
+    p A Control is a clickable, interactive image within a scene. #[a(href='#participants') Participants] can interact with the control using keyboard, mouse, or controller. There are currently has two types of controls - Buttons and Joysticks. Additional control types may be added in the future.
```

Comment:
"image" is the wrong language here. As is "clickable" interactive controls will eventually work with keyboard, mouse, xbox control, touch(mobile phones/tablets)"

Also are dashes(-) usually used to separate clauses like this? A colon might be better?